basic-examples/expressions/config.yaml

```diff
@@ -0,0 +1,8 @@
+deployers:
+  image:
+    deployer_name: docker
+log:
+  level: debug    
+logged_outputs:
+  error:
+    level: error
```

Comment:
I really do always prefer trailing newlines, and so does GitHub. (Although perhaps it just *loves* to display that annoying red circle...)